setup.cfg

```diff
@@ -75,7 +75,7 @@ install_requires =
     aiohttp-retry>=2.4.5
     diskcache>=5.2.1
     jaraco.windows>=5.7.0; python_version < '3.8' and sys_platform == 'win32'
-    scmrepo==0.0.10
+    scmrepo>=0.0.11
```

Comment:
I'd prefer this to be pinned so that older DVC versions still work even if we make any incompatible changes to the `scmrepo` in the future.

Reply:
Yeah, I was actually going to ask if you thought we should start doing something like `scmrepo<0.1.0,>=0.0.11` instead of pinning a specific release, with the assumption that the next time we add some incompatible changes (like the relpath change) in scmrepo we would go ahead and bump the minor version there instead of the patch version.

@skshetry